kotlinx-coroutines-core/common/src/flow/internal/Combine.kt

```diff
@@ -54,7 +54,8 @@ internal suspend fun <R, T> FlowCollector<R>.combineInternal(
         ++currentEpoch
         // Start batch
         // The very first receive in epoch should be suspending
-        var element = resultChannel.receiveOrNull() ?: break // Channel is closed, nothing to do here
+        // TODO closed with an exception ?
```

Comment:
`resultChannel` cannot be closed with an exception

Reply:
Thanks, checked it, but forgot to remove my internal TODO